main/lsp/ErrorReporter.cc

```diff
@@ -0,0 +1,91 @@
+#include "main/lsp/ErrorReporter.h"
+#include "core/lsp/TypecheckEpochManager.h"
+#include "main/lsp/LSPMessage.h"
+#include "main/lsp/LSPOutput.h"
+#include "main/lsp/json_types.h"
+
+namespace sorbet::realmain::lsp {
+using namespace std;
+ErrorReporter::ErrorReporter(shared_ptr<const LSPConfiguration> config) : config(config) {}
+
+// Used for unit tests
```

Comment:
Documentation for methods should go in the header file. That's where people will look to see what methods are available on an object. clangd will display comments in header files in autocomplete, too.